### PR TITLE
msstitch version to 2.13

### PR DESCRIPTION
**Pull request by glormph**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [X] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/msstitch/meta.yaml [hunk 1]
```diff
@@ -18,14 +18,14 @@ build:
 
 requirements:
   host:
-    - python
-    - setuptools
+    - python >3
```
Comment (dpryan79) [inline recipes/msstitch/meta.yaml]: There are no versions of python >3

File: recipes/msstitch/meta.yaml [hunk 2]
```diff
@@ -18,14 +18,14 @@ build:
...
+    - pip
     - numpy
     - lxml
     - biopython >=1.69
     - pyyaml
```
Comment (dpryan79) [inline recipes/msstitch/meta.yaml]: Can you remove everything except python and pip from the `host:` section?

File: recipes/msstitch/meta.yaml (hunk 1 above)
Comment (glormph) [reply recipes/msstitch/meta.yaml]: Is that not what it is supposed to look like according to: https://bioconda.github.io/linting.html#noarch-or-not-noarch ? I assumed it was to be above 3.0.

**Comment by npavlovikj** [comment]
@BiocondaBot recheck

File: recipes/msstitch/meta.yaml (hunk 1 above)
Comment (dpryan79) [reply recipes/msstitch/meta.yaml]: That's a typo in the documentation, it should be `>=3`.

Comment (glormph) [reply recipes/msstitch/meta.yaml]: No problem, thank you :)

**Comment by dpryan79** [comment]
@bioconda-bot recheck

**Comment by dpryan79** [comment]
:+1:

**Comment by dpryan79** [comment]
@bioconda-bot merge

**Pull request merged**